src/main/java/org/apache/maven/shared/dependency/analyzer/asm/DependencyClassFileVisitor.java

```diff
@@ -56,7 +56,7 @@ public void visitClass( String className, InputStream in )
     {
         try
         {
-            byte[] byteCode = IOUtils.toByteArray( in );
+            byte[] byteCode = IOUtil.toByteArray( in );
```

Comment:
This can also be done with NIO2. No need for P U

Reply:
It is InputStream ... 
plexus utils is used anyway in other place. This was easy replace, allowing to reduce one dependency from compile scope